File: ios/Switch.m
//
//  Switch.m
//  documentDeneme
//
//  Created by Alper Salık on 7.02.2022.
//

#import <Foundation/Foundation.h>
#import "React/RCTViewManager.h"
@interface RCT_EXTERN_MODULE(Switch, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(isOn  , BOOL)
@end

File: ios/Switch.swift
//
//  Switch.swift
//  documentDeneme
//
//  Created by Alper Salık on 7.02.2022.
//

import Foundation
import UIKit

@objc(Switch)
class Switch: RCTViewManager {
  
  override func view() -> UIView! {
    return SwitchView().view
  }
  
  override static func requiresMainQueueSetup() -> Bool {
    return true
  }
}
